docs/index.md

```diff
@@ -35,6 +36,22 @@ Additionally, we prioritise the security of the data during its processing and t
 
 ![](_static/structure.png)
 
+### Contributions
+
+Besides our [models](https://www.foundationmodels.dk/models/). The Danish Foundation models project have led to contributions for a series of open-source projects, the following table include some of these contributions:
+
+
+| Project                                                                                                                | Contribution                                                  |
+| ---------------------------------------------------------------------------------------------------------------------- | ------------------------------------------------------------- |
+| **Packages**                                                                                                           |                                                               |
+| [NLPDedup](https://github.com/saattrupdan/NLPDedup)                                                                    | A deduplication library derived from DFM's deduplication code |
+| **Code contributions**                                                                                                 |                                                               |
+| [TextDescriptives](https://hlasse.github.io/TextDescriptives/)                                                         | Added heuristic quality measure for texts                     |
+| [dolma](https://github.com/allenai/dolma)                                                                              | Bugfixes and addition of taggers for filtering                |
+| **Datasets**                                                                                                           |                                                               |
+| [dagw_reddit_filtered_v1.0.0](https://huggingface.co/datasets/DDSC/dagw_reddit_filtered_v1.0.0)                        | A filtered version of the Danish Gigaword, including reddit   |
```

Comment:
Add the machine translated multiple choice datasets?

1. https://huggingface.co/datasets/alexandrainst/m_arc
2. https://huggingface.co/datasets/alexandrainst/m_mmlu
3. https://huggingface.co/datasets/alexandrainst/m_hellaswag
4. https://huggingface.co/datasets/alexandrainst/m_truthfulqa

Reply:
Yep, def. add these as well